syntax = "proto3";

package verification;

import "common.proto";

service VerificationService {
    rpc SayVerification (common.Request) returns (TransactionResponse);
    rpc initVerification (common.InitAllInfoRequest) returns (common.Empty);
    rpc BookListNotEmtpy (common.Request) returns (common.Response);
}

message InitRequest {
    string order_id = 1;
    common.AllInfoRequest transaction_request = 2;
}


message TransactionResponse {
    bool is_verified = 1;
    string message = 2;
}
